scw/errors.go

```diff
@@ -256,20 +256,17 @@ func (e *OutOfStockError) GetRawBody() json.RawMessage {
 	return e.RawBody
 }
 
-type clientCredentialErrorType string
-
-const (
-	clientCredentialError_EmptyAccessKey = clientCredentialErrorType("access key cannot be empty")
-	clientCredentialError_EmptySecreyKey = clientCredentialErrorType("secret key cannot be empty")
-)
+// ClientValidationError indicates that at least one of client data have been badly provided for the client creation.
+type ClientValidationError struct {
```

Comment:
what about `InvalidClientOptionError` ?

Reply:
"Invalid" and "Error" in the same name? 🤔

Reply:
Yes : Client option is Invalid  and this is an error

In Java or PHP class name are `InvalidParameterException` for example